src/main.rs

```diff
@@ -78,7 +78,9 @@ fn create_app<'a, 'b>(
     api_server_path: &'a str,
 ) -> App<'a, 'b> {
     App::new("cloud-hypervisor")
-        .version(crate_version!())
+        // 'BUILT_VERSION' is set by the build script 'build.rs' at
+        // compile time
+        .version(env!("BUILT_VERSION"))
```

Comment:
Nit: I would prefer `CLH_COMIMIT_ID` unless want to use for other purposes

Reply:
@jcvenegas For now, the output of `$ cloud-hypervisor  --version` is '`cloud-hypervisor v0.5.0-40-gba01c64`'. I thought it is almost identical to `crate_version!()+CLH_COMMIT_ID` except it contains the number of additional commits on top of the latest tag (a.k.a the latest crate version). Please advice if I miss anything. Thanks.

Reply:
ups my bad, I just tested and is fine, I thought was only the commit

Reply:
Great. Glad it aligns with what you want. :)